torch/nn/quantized/modules/linear.py

```diff
@@ -146,18 +140,20 @@ def forward(self, x):
     # from the QTensor weight.
     def _save_to_state_dict(self, destination, prefix, keep_vars):
         super(Linear, self)._save_to_state_dict(destination, prefix, keep_vars)
-        destination[prefix + 'weight'] = self.weight()
+        (w, b) = self.weight()
```

Comment:
Can we have a seperate function self.bias() to obtain the bias instead? This way the function names map to individual attributes.

Reply:
@dzhulgakov suggested to update existing `weight()` function to return bias as well. Maybe I can rename it to weight_bias() instead?